pkgs/applications/science/logic/coq/default.nix

```diff
@@ -70,9 +70,9 @@ let
       { case = range "8.7" "8.10";  out = ocamlPackages_4_09; }
       { case = range "8.5" "8.6";   out = ocamlPackages_4_05; }
     ] ocamlPackages_4_12;
-  ocamlNativeBuildInputs = [ ocamlPackages.ocaml ocamlPackages.findlib ]
+  ocamlNativeBuildInputs = [ ocamlPackages.ocaml ]
     ++ optional (coqAtLeast "8.14") ocamlPackages.dune_2;
-  ocamlBuildInputs = []
+  ocamlBuildInputs = [ ocamlPackages.findlib ]
```

Comment:
`findlib` is a executable that should be in `nativeBuildInputs` otherwise you'll break `useStrict`

Reply:
With this PR, `ocamlBuildInputs` are now propagated, so this should rather be `propagatedNativeBuildInputs` I suppose.

Reply:
fixed